### ISSUE TITLE
Possible wrong packaging of the release downloads (zip, tar.gz) since version 2.5.0

### ISSUE THREAD

**Issue by m-kotowski** [e-Spirit/FSDevTools#29]
Hi team.

Before version 2.5.0 you provided the Windows fs-cli.cmd file in the downloadable zip file, and the Linux/Mac fs-cli file in the downloadable tar.gz file.

Since version 2.5.0 both downloadable archives contain fs-cli.cmd and **fs-cli.sh** (with `.sh` extension), but the documentation still talks about fs-cli without e.g. `.sh` extension. Therefore, I assume that there is an issue with your packaging tool.

Furthermore, before version 2.5.0 the fs-cli script was provided with execute rights, since version 2.5.0 the fs-cli.sh has no execute rights anymore. Is a minor issue, but it would be nice if you could provide the scripts again with execute rights.

In addition, the end-of-line style of fs-cli.sh seems to be Windows style instead of Linux style, so I get the following error message while running `sh fs-cli.sh`:

```bash
$ sh ./fs-cli.sh
: not found: 2: ./fs-cli.sh: 
: not found: 3: ./fs-cli.sh: 

: not found: 7: ./fs-cli.sh: }
: not found: 8: ./fs-cli.sh: 
: not found: 5: ./fs-cli.sh: {
Fehler: Hauptklasse com.espirit.moddev.cli.Main konnte nicht gefunden oder geladen werden
Ursache: java.lang.ClassNotFoundException: com.espirit.moddev.cli.Main
: not found: 10: ./fs-cli.sh: 
: not found: 11: ./fs-cli.sh: 
```

After applying the execute right, I encounter this error message indicating that `^M` is part of the line ending.

```bash
$ chmod +x fs-cli.sh
$ ./fs-cli.sh
bash: ./fs-cli.sh: /bin/sh^M: Defekter Interpreter: Datei oder Verzeichnis nicht gefunden
```

After running the following command, I can execute fs-cli.sh as usual:

```bash
sed -i 's/\r//' fs-cli.sh
```

Finally, I noticed that you changed the archive naming scheme (breaking my tools ;) ) - since version 2.5.0 the version is part of the archive names. Is this intentional?

Regards, Michael.

**Comment by kafmann-e-spirit** [comment]
Hello Michael, 
thanks for this issue. We will investigate the issue and fix it with one of the next versions.

I'm trying to answer some of your quesions here:

- Talking about the documentation (fs-cli without .sh extension):
This is currently intended, because we don't want to create different documentations for different operating systems. Thats the reason why there is no extension included in the documentation.

- Talking about the unexecuteable .sh-file:
This seems to be a bug, we will fix it. Thanks :)

- Talking about the naming scheme:
The change of the naming scheme was intentional, because we want to stick to the maven-versioning-naming-scheme. Thats the reason why we did append the version to the release-artifacts. The inner jar files (in lib/) still have the same name as before 2.5.0.

I hope the answers helped you. If you have some more questions, feel free to ask :)

**Comment by kafmann-e-spirit** [comment]
Hello Michael,

this issue has been fixed with [Release 2.5.2](https://github.com/e-Spirit/FSDevTools/releases/tag/2.5.2).

Thanks for reporting this issue!
